test/testUtils/helpers/overridesHelper.ts

```diff
@@ -332,6 +332,17 @@ export class OverridesHelper extends GameManagerHelper {
     return this;
   }
 
+  /**
+   * Overrides the enemy (pokemon)'s IVs
+   * @param ivs an array of 6 numbers ranging from 1-31
+   * @returns
+   */
+  public enemyIVs(ivs: number[]): this {
```

Comment:
```suggestion
  public enemyIVs(ivs: number ! number[]): this {
```
The override allows this, so might as well have it too